data/armies/skaven.json

```
@@ -4,7 +4,7 @@
     {
       "name": "Arch-Warlock",
       "size": "32 mm",
-      "move": "",
+      "move": "a",
```

I'm guessing this was a typo
```suggestion
      "move": "",
```